tests/aws/services/lambda_/test_lambda_api.snapshot.json

```diff
@@ -5215,26 +5479,18 @@
         }
       },
       "list_code_signing_configs": {
-        "CodeSigningConfigs": [
-          {
-            "AllowedPublishers": {
-              "SigningProfileVersionArns": [
-                "arn:aws:signer:<region>:111111111111:/signing-profiles/test"
-              ]
-            },
-            "CodeSigningConfigArn": "arn:aws:lambda:<region>:111111111111:code-signing-config:<resource:1>",
-            "CodeSigningConfigId": "<resource:1>",
-            "CodeSigningPolicies": {
-              "UntrustedArtifactOnDeployment": "Warn"
-            },
-            "Description": "Testing CodeSigning Config",
-            "LastModified": "date"
-          }
-        ],
-        "ResponseMetadata": {
-          "HTTPHeaders": {},
-          "HTTPStatusCode": 200
-        }
+        "AllowedPublishers": {
```

Comment:
@dfangl FYI: AWS apparently changed the code signing behavior (see [this backlog item](https://www.notion.so/localstack/Improve-parity-with-Lambda-Code-Signing-Config-6a8432e18105414aad07502023a6f279?pvs=4))